Sources/NIOSSL/SSLContext.swift

```diff
@@ -159,10 +159,26 @@ public final class NIOSSLContext {
         returnCode = CNIOBoringSSL_SSL_CTX_set_cipher_list(context, configuration.cipherSuites)
         precondition(1 == returnCode)
 
+        // On non-Linux platforms, when using the platform default trust roots, we make use of a
+        // custom verify callback. If we have also been presented with additional trust roots of
+        // type `.file`, we take the opportunity now to load them in memory to avoid doing so
+        // repeatedly on the request path.
+        //
+        // However, to avoid closely coupling this code with other parts (e.g. the platform-specific
+        // concerns, and the defauling of `trustRoots` to `.default` when `nil`), we unilaterally
```

Comment:
Nit: defaulting

```suggestion
        // concerns, and the defaulting of `trustRoots` to `.default` when `nil`), we unilaterally
```

Reply:
Addressed in https://github.com/apple/swift-nio-ssl/pull/284/commits/0bae5eae1f6588c0d8d9a0ea6269454ea4194581.